examples/AdminClient/Program.cs

```diff
@@ -583,27 +583,49 @@ static async Task ListConsumerGroupsAsync(string bootstrapServers, string[] comm
         {
             var timeout = TimeSpan.FromSeconds(30);
             var statesList = new List<ConsumerGroupState>();
-            try
+            var groupTypesList = new List<ConsumerGroupType>();
+            var isType = false;
+            var isState = false;
+            for (int i = 0; i < commandArgs.Length; i++)
             {
-                if (commandArgs.Length > 0)
+                if (commandArgs[i] == "-states")
                 {
-                    timeout = TimeSpan.FromSeconds(Int32.Parse(commandArgs[0]));
+                    if (isState)
+                    {
+                        Console.WriteLine("usage: .. <bootstrapServers> list-consumer-groups [-states <match_state_1> <match_state_2> ... <match_state_N>] [-types <group_type_1> .. <group_type_M>]");
+                        Environment.ExitCode = 1;
+                        return;
+                    }
+                    isState = true;
                 }
-                if (commandArgs.Length > 1)
+                else if (commandArgs[i] == "-types")
+                {
+                    if (isType)
+                    {
+                        Console.WriteLine("usage: .. <bootstrapServers> list-consumer-groups [-states <match_state_1> <match_state_2> ... <match_state_N>] [-types <group_type_1> .. <group_type_M>]");
+                        Environment.ExitCode = 1;
+                        return;
+                    }
+                    isType = true;
+                }
+                else
                 {
-                    for (int i = 1; i < commandArgs.Length; i++)
+                    if (isState)
                     {
                         statesList.Add(Enum.Parse<ConsumerGroupState>(commandArgs[i]));
                     }
+                    else if (isType)
+                    {
+                        groupTypesList.Add(Enum.Parse<ConsumerGroupType>(commandArgs[i]));
+                    }
+                    else
+                    {
+                        Console.WriteLine("usage: .. <bootstrapServers> list-consumer-groups [-states <match_state_1> <match_state_2> ... <match_state_N>] [-types <group_type_1> .. <group_type_M>]");
```

Comment:
Missing `<timeout_seconds>`

Reply:
None of the usage in codebase mentions timeout seconds.

Reply:
For the examples convention I am saying

Reply:
See the remove line 602 it had `< timeout_seconds>`